src/views/Validator/Validator.tsx

```diff
@@ -155,6 +163,16 @@ export function Validator() {
           />
         }
       />
+      {scriptPath !== undefined && (
```

Comment:
I'm not entirely sure why `scriptPath` would ever be `undefined`. If you're viewing a script in the validator shouldn't it always have a path?

Reply:
If you open the validator from the home page it will be empty as you have to select a script

Reply:
I see. 

The UX could definitely improve there. It's not very helpful that I'm dropped in to an empty view where the only indication of what I can use it for is to import an external script (which we currently have pretty poor support for).

The view should be more guiding. Ask the user to select a script, encourage them to generate scripts with a generator and allow them to import a script. It would be the validator equivalent of the start recording page, which doesn't just show you an empty recording. It acts as a bit of a guide to start a recording.